addons/mail/static/src/components/mobile_messaging_navbar/mobile_messaging_navbar.js

```diff
@@ -28,6 +30,10 @@ Object.assign(MobileMessagingNavbar, {
     },
     props: {
         activeTabId: String,
+        onSelectMobileMessagingNavbarTab: {
+            type: Function,
+            optional: true,
```

Comment:
required?

Reply:
I'm not fond in enforcing `required` on case by case of current use, especially when these props are just for hooking behavior from the parent.